src/main/java/org/sunbird/profile/service/ProfileServiceImpl.java

```diff
@@ -177,11 +173,17 @@ public SBApiResponse profileUpdate(Map<String, Object> request, String userToken
 						getModifiedPersonalDetails(profileDetailsMap.get(changedObj), requestData);
 					}
 				}
+				log.info("Existing Profile Details : " + new Gson().toJson(existingProfileDetails));
+				String schema = getVerifiedProfileSchema();
+				if (validateJsonAgainstSchema(schema, new Gson().toJson(existingProfileDetails))) {
```

Comment:
toJson -- is this throws error ?? if so, just pass the existingProfileDetails to the method and convert it to json inside a try catch in that method...